packages/shared/yup-validation-resolver/useYupValidationResolver.ts

```diff
@@ -29,7 +38,7 @@ const useYupValidationResolver = <T>(validationSchema: AnySchema): Resolver =>
                 ...allErrors,
                 [currentError.path]: {
                   type: currentError.type ?? 'validation',
-                  message: currentError.message,
+                  message: t(currentError.message),
```

Comment:
this will not work like this... if u need translations make sure "currentError.message" is itself translated at the time when we are initialising "currentError" object with the error messages...
```suggestion
                  message: currentError.message,
```

Reply:
which mu assumption is we are already doing `fieldRequirementsTranslations.maxChars(t, 43),` when we are passing `t` functions to the error messages, right ??

Reply:
When a required error is raised the ```currentError.message``` value will be "Required" which is a key already set in ```locales/en/plugin__odf-console.json```.

This is implemented like to avoid repeating ```t('Required')``` multiple times on every single form